cmds/upspin_sos/server.go

```diff
@@ -203,6 +203,7 @@ func (us *UpspinServer) buildRouter() *mux.Router {
 	r.HandleFunc("/", us.displayStateHandle).Methods("GET")
 	r.HandleFunc("/edit", us.editHandle).Methods("POST")
 	r.HandleFunc("/submit", us.submitHandle).Methods("POST")
+	sos.SetHTMLPath([]string{"/etc/sos/html"})
```

Comment:
so, when we talked about this we wanted to define it like so:
SetHTMLPath(n ...string)
and then in the calls you could do stuff like this:
sos.SetHTMLPath("a", "b")
or even 
sos.SetHTMLPath()
which seems a lot more convenient.
I think you should change SetHTMLPath to either
SetHTMLPath(s string)
or 
SertHTMLPath(s ...string)
but not what you've got now. Calling it is pretty inconvenient the way you've defined it.